### ISSUE TITLE
Phyiscs collision crash when entity is despawned after colliding

### ISSUE THREAD

**Issue by AnthonyTornetta** [AnthonyTornetta/Cosmos#282]
```
thread 'Compute Task Pool (0)' panicked at ./bevy_rapier3d/../src/pipeline/events.rs:71:14:
Internal error: entity not found for collision event.
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Encountered a panic in system `bevy_rapier3d::plugin::systems::step_simulation<cosmos_core::physics::collision_handling::CosmosPhysicsFilter>`!
Encountered a panic in system `bevy_app::main_schedule::Main::run_main`!
```

**Comment by AnthonyTornetta** [comment]
Linked to physics engine update, will probably be fixed after #275

**Comment by AnthonyTornetta** [comment]
Closing since #275 has been put into main. Will reopen if issue persists.
